external/fv3fit/fv3fit/pytorch/cyclegan/cyclegan_trainer.py

```diff
@@ -447,25 +447,29 @@ def __post_init__(self):
     def _call_generator_a_to_b(self, input):
         if self._script_gen_a_to_b is None:
             self._script_gen_a_to_b = torch.jit.trace(
-                self.generator_a_to_b.forward, input
+                self.generator_a_to_b.forward, (input,)
```

Comment:
At some point the call signature of `torch.jit.trace` was updated from taking a tensor argument to taking a tuple of tensors, to allow support for objects that take multiple arguments. If you pass a tensor instead of a tuple of tensors, it automatically gets converted into a tuple containing one tensor. I updated this to be consistent with the new API, as I was using some forwards taking multiple arguments at one point in my working branch.

I will definitely type-hint this function, I have that change in my working branch and plan to pull it into another PR.